src/frequenz/client/common/microgrid/__init__.py

```diff
@@ -1,4 +1,51 @@
 # License: MIT
-# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
 
-"""Frequenz microgrid definition."""
+"""Module to define the microgrid used with the common client."""
+
+from __future__ import annotations  # required for constructor type hinting
+
+from dataclasses import dataclass
+from typing import List, Self
+
+# pylint: disable=import-error, no-name-in-module
+from frequenz.api.common.v1.microgrid.microgrid_pb2 import (
+    MicrogridComponentIDs as PBMicrogridComponentIDs,
+)
+
```

Comment:
Even when is the last import, I would still re-enable the check, just in case more imports are added in the future.

```suggestion
# pylint: enable=no-name-in-module
```